example-app/src/screens/HomeScreen.tsx

```diff
@@ -205,8 +205,6 @@ function mapFromDiscoveryMethod(method: Reader.DiscoveryMethod) {
       return 'Bluetooth Proximity';
     case 'internet':
       return 'Internet';
-    case 'embedded':
-      return 'Embedded';
```

Comment:
@bric-stripe this was failing typescript, we also don't have any support for connecting to an embedded reader anyways so removing